site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -628,6 +629,12 @@ public function showStatus($gradeable_id) {
                 }
                 $total_submissions += $value;
             }
+            foreach ($total_users_who_submitted as $key => $value) {
+                if ($key == 'NULL') {
```

Comment:
Always better to use the `===` operator in PHP (and JS) instead of `==` as the latter is a weak form of equivalence vs `===` which is strong equivalence (ensures things like types are equivalent and does not do implicit type coercion).

```suggestion
                if ($key === 'NULL') {
```